### PR TITLE
RATEST-225: Adding method for deleting a condition on Conditions Page

### PR DESCRIPTION
**Pull request by Ndacyayisenga-droid**
Adding missing methods due to  https://github.com/openmrs/openmrs-contrib-qaframework/pull/137

cc @kdaud  @dkayiwa @sherrif10

### CODE REVIEW

File: ui-tests/src/test/java/org/openmrs/reference/page/ConditionsPage.java [hunk 1]
```diff
@@ -18,6 +18,7 @@
     private static final By EDIT = By.cssSelector("i[title='Edit Condition: ']");
     private static final By DELETE = By.cssSelector("i[title='Delete']");
     private static final By CONDITIONS_LIST = By.cssSelector("tr.clickable-tr");
+    private static final By DELETE_CONDITION = By.cssSelector("#remove-condition-dialog > div.dialog-content > button.confirm.right");
```
Comment (kdaud) [inline ui-tests/src/test/java/org/openmrs/reference/page/ConditionsPage.java:21]: @Ndacyayisenga-droid  what do you think of `CONFIRM_DELETE_CONDITION`?

Comment (Ndacyayisenga-droid) [reply ui-tests/src/test/java/org/openmrs/reference/page/ConditionsPage.java]: > @Ndacyayisenga-droid what do you think of `CONFIRM_DELETE_CONDITION`?

@kdaud done

**Review by kdaud — APPROVED** [review]
@sherrif10 what do yo have to ask about this PR?

**Comment by kdaud** [comment]
@Ndacyayisenga-droid its a good idea to rename the commit message to some thing more readable before we can have it merged.

**Comment by Ndacyayisenga-droid** [comment]
> @Ndacyayisenga-droid its a good idea to rename the commit message to some thing more readable before we can have it merged.

@kdaud  does it look nice nice now?

**Comment by kdaud** [comment]
> does it look nice nice now?

Where is the change you have made?

**Comment by Ndacyayisenga-droid** [comment]
> > does it look nice nice now?
> 
> Where is the change you have made? You may need to take a second look at the comment [above](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/572#issuecomment-927990692)?

@kdaud  did u see [this](Adding functionality for deleting conditions)

File: ui-tests/src/test/java/org/openmrs/reference/page/ConditionsPage.java [hunk 2]
```diff
@@ -89,4 +90,8 @@ public ConditionPage clickOnAddNewCondition() {
     public List<WebElement> getConditionsList() {
         return findElements(CONDITIONS_LIST);
     }
+
+    public void confirmDeleteCondition2(){
```
[the remark this answers is not in the document]
Comment (Ndacyayisenga-droid) [reply ui-tests/src/test/java/org/openmrs/reference/page/ConditionsPage.java]: > @Ndacyayisenga-droid what do you have to say about `confirmDeleteCondition()`?

@kdaud done

**Comment by kdaud** [comment]
@dkayiwa we can have this commit merged!

File: ui-tests/src/test/java/org/openmrs/reference/page/ConditionsPage.java [hunk 3]
```diff
@@ -18,6 +18,7 @@
     private static final By EDIT = By.cssSelector("i[title='Edit Condition: ']");
     private static final By DELETE = By.cssSelector("i[title='Delete']");
     private static final By CONDITIONS_LIST = By.cssSelector("tr.clickable-tr");
+    private static final By CONFIRM_DELETE_CONDITION = By.cssSelector("#remove-condition-dialog > div.dialog-content > button.confirm.right");
```
Comment (dkayiwa) [inline ui-tests/src/test/java/org/openmrs/reference/page/ConditionsPage.java:21]: Is this a link, label of what?

Comment (Ndacyayisenga-droid) [reply ui-tests/src/test/java/org/openmrs/reference/page/ConditionsPage.java:21]: > Is this a link, label or what?

@dkayiwa Its a `Button`  but unfournately it has no id

Comment (Ndacyayisenga-droid) [reply ui-tests/src/test/java/org/openmrs/reference/page/ConditionsPage.java:21]: > Is this a link, label or what?

@dkayiwa Its a `Button`  but unfournately it has no id

Comment (dkayiwa) [reply ui-tests/src/test/java/org/openmrs/reference/page/ConditionsPage.java:21]: How do i tell so from its name?

Comment (Ndacyayisenga-droid) [reply ui-tests/src/test/java/org/openmrs/reference/page/ConditionsPage.java:21]: > How do i tell so from its name?

@dkayiwa should it be something like this `CONFIRM_DELETE_CONDITION_BUTTON`

Comment (Ndacyayisenga-droid) [reply ui-tests/src/test/java/org/openmrs/reference/page/ConditionsPage.java]: > How do i tell so from its name?

@dkayiwa  done

**Comment by Ndacyayisenga-droid** [comment]
Thanks @dkayiwa  I think we can now take troouble to see whats happening here https://github.com/openmrs/openmrs-contrib-qaframework/pull/137

**Pull request merged**